docs/source/_templates/home.html

```diff
@@ -697,10 +697,18 @@ <h1 class="gallery-title">Features</h1>
                             <h5>About</h5>
                         </div>
                         <ul class="link-list">
-                            <li class="link-list"><a class="footer-link" href="community.html#contributors"><i class="fa fa-angle-double-right"></i>Developers</a></li>
-                            <li class="link-list"><a class="footer-link" href="https://discord.gg/6btFPPj"><i class="fa fa-angle-double-right"></i>Support</a></li>
-                            <li class="link-list"><a class="footer-link" href="installation.html"><i class="fa fa-angle-double-right"></i>Download</a></li>
-                            <li class="link-list"><a class="footer-link" href="auto_tutorials/index.html"><i class="fa fa-angle-double-right"></i>Tutorials</a></li>
+                            <li class="link-list"><a class="footer-link" href="community.html#contributors"><i class="fa fa-angle-double-right"></i> 
```

Comment:
Giving space between the double arrow and text in the footer makes the layout consistent all over